bin/i18n/hoc_sync_utils.rb

```diff
@@ -33,27 +33,24 @@ def self.sync_in
   end
 
   def self.sync_out
-    rename_downloads_from_crowdin_code_to_locale
+    rename_yml_to_locale
     copy_from_i18n_source_to_hoc
     restore_sanitized_headers
   end
 
-  def self.rename_downloads_from_crowdin_code_to_locale
+  def self.rename_yml_to_locale
     puts "Updating crowdin codes to our locale codes..."
     Languages.get_hoc_languages.each do |prop|
       # move downloaded folders to root source directory and rename from
       # language to locale
-      crowdin_dir = File.join(I18N_SOURCE_DIR, "hourofcode", prop[:crowdin_name_s])
       dest_dir = "i18n/locales/#{prop[:locale_s]}"
-      next unless File.directory?(crowdin_dir)
-
-      FileUtils.cp_r File.join(crowdin_dir, '.'), dest_dir
-      FileUtils.rm_r crowdin_dir
```

Comment:
Don't we still need this operation, and also the "remove unused" one below? Before, translations would get downloaded to (for example) `/i18n/locales/source/hourofcode/Spanish/` and now they'll be downloaded to `/i18n/locales/Spanish/hourofcode/`; we _want_ them to be in `/i18n/locales/es-MX/hourofcode/`; the work needed hasn't changed, just the specific location of the source.

Reply:
But the `sync-out` does that now.

Reply:
Specifically this function now also copies the `hourofcode` downloaded files: https://github.com/code-dot-org/code-dot-org/blob/1058958f4202fd192c2b4912cfd11933b0352c54/bin/i18n/sync-out.rb#L31

Reply:
ooooooh of course. I forgot to consider that these syncs are unified now